gwt-geolocation-j2cl-tests/pom.xml

```diff
@@ -1,100 +1,102 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
-    <modelVersion>4.0.0</modelVersion>
+  <modelVersion>4.0.0</modelVersion>
 
-    <parent>
-        <groupId>org.gwtproject.geolocation</groupId>
-        <artifactId>gwt-geolocation-parent</artifactId>
-        <version>1.0-SNAPSHOT</version>
-    </parent>
-    <artifactId>gwt-geolocation-j2cl-tests</artifactId>
+  <parent>
+    <groupId>org.gwtproject.geolocation</groupId>
+    <artifactId>gwt-geolocation-parent</artifactId>
+    <version>dev</version>
+  </parent>
+  <artifactId>gwt-geolocation-j2cl-tests</artifactId>
+  <version>${revision}</version>
 
-    <name>GWT Geo-Location J2CL Tests</name>
-    <description>Test cases for the J2Cl tests</description>
-    <url>https://github.com/gwtproject/gwt-geolocation</url>
+  <name>GWT Geo-Location J2CL Tests</name>
+  <description>Test cases for the J2Cl tests</description>
+  <url>https://github.com/gwtproject/gwt-geolocation</url>
 
-    <properties>
-        <maven.j2cl.plugin>0.13-SNAPSHOT</maven.j2cl.plugin>
-        <maven.surfire.plugin>3.0.0-M1</maven.surfire.plugin>
-        <!-- CI -->
-        <vertispan.j2cl.repo.url>https://repo.vertispan.com/j2cl/</vertispan.j2cl.repo.url>
-        <jsinterop.base.version>1.0.0-SNAPSHOT</jsinterop.base.version>
-        <j2cl.version>0.7-SNAPSHOT</j2cl.version>
-        <slf4j.version>1.7.5</slf4j.version>
-    </properties>
+  <properties>
+    <maven.j2cl.plugin>0.16-SNAPSHOT</maven.j2cl.plugin>
+    <maven.surfire.plugin>3.0.0-M1</maven.surfire.plugin>
+    <!-- CI -->
+    <vertispan.j2cl.repo.url>https://repo.vertispan.com/j2cl/</vertispan.j2cl.repo.url>
+    <jsinterop.base.version>1.0.0-SNAPSHOT</jsinterop.base.version>
```

Comment:
should be 1.0.0

Reply:
actually i dont think you need this at all

Reply:
Oh yes, you are right. Not needed.